con/2018/events.json

```diff
@@ -3,21 +3,21 @@
     {
         "date": "5",
         "day": "Wednesday",
-        "description" : "Training & Hackathon",

```

Comment:
_Training_ is [usually](https://www.oxfordlearnersdictionaries.com/definition/english/training) uncountable noun, though "Trainings" is pretty popular. Kepp or change it as you think.